ui/src/app/navigation/abstract-search.component.ts

```diff
@@ -49,4 +50,8 @@ export class AbstractSearchComponent {
     return false
   }
 
+  isDisabled(path: string): boolean {
```

Comment:
I suggest positive naming here.

Reply:
Devon, what do you recommend? The html property is disabled. You good with "isEnabled"?

Reply:
This isn't crucial, obviously.  And I've probably violated my own suggestions countless times...

Yes, I recommend isEnabled.  I can see the motivation to be consistent with the html property name, but it would be consistent with a bad practice.  ;)

> Use positive boolean variable names Negative names like notFound, notdone, and notSuccessful are difficult to read when they are negated...

Reference: [11.2. Naming Specific Types of Data](https://flylib.com/books/en/2.823.1.100/1/)

Reply:
Done deal.